### ISSUE TITLE
[J9][Wizard] Surefire UI Tests dont run

### ISSUE THREAD

**Issue by cdietrich** [eclipse/xtext-core#605]
something like 
```
<tycho.testArgLine>-XstartOnFirstThread --add-modules=ALL-SYSTEM</tycho.testArgLine>
```
helps but i am not sure if this should not be fixed in tycho itself
https://dev.eclipse.org/mhonarc/lists/tycho-user/msg07844.html

**Comment by cdietrich** [comment]
If we have to take the all-modules path then we have to make sure its active only if J9 or later is selected in the wizard cause of `Unrecognized option: --add-modules=ALL-SYSTEM`

**Comment by cdietrich** [comment]
The Tycho guys say they prefer not to fix this in tycho so we need to go the property way

**Comment by cdietrich** [comment]
Fixed in 2.14
